scripts/install_skills.py

```
@@ -0,0 +1,322 @@
+                sys.stderr.write(f"  {c}\n")
+            if len(plan.collisions) > 10:
+                sys.stderr.write(f"  ... and {len(plan.collisions) - 10} more\n")
+        return 1
+
+    if args.dry_run:
+        if not args.json:
+            sys.stdout.write(
+                f"dry run: {len(plan.actions)} artifact(s) -> {args.target_dir} "
+                f"(layout={args.layout})\n"
+            )
+            for artifact, dest in plan.actions[:20]:
```

Ruff (0.15.13)

[warning] Loop control variable `dest` not used within loop body

Rename unused `dest` to `_dest`

(B007)